src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -27,4 +27,46 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    @Override
+    public boolean equals(Object object) {
+        System.out.println(width + " -- " + length + " -- " + color);
+
+        if (this == object) {
+            return true;
+        }
+
+        if (null == object) {
+            return false;
+        }
+
+        if (object instanceof Rectangle) {
```

Comment:
what is the difference between instanceof and getClass()?

Reply:
Fixed. Done